bin/MadGraph5_aMCatNLO/cards/production/14TeV/tHbj_TL_M1000GeV_W10p/tHbj_TL_M1000GeV_W10p_proc_card.dat

```diff
@@ -0,0 +1,19 @@
+sset group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+ ta+ 
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define al = j l+ l- vl vl~
```

Comment:
is "al" a typo and you wanted to write "all" ?
where are you using tt and ww? please remove them if not used

Reply:
The al, tt, ww definitions are though not used further, but there is no harm in leaving them like this. The file is just a copy of 13 TeV sample. https://github.com/dmajumder/genproductions/blob/5adf381d9e6ab8e6c3d7da4aa7f4370114f1b142/bin/MadGraph5_aMCatNLO/cards/production/13TeV/tHbj_TL_M1000GeV_W10p/tHbj_TL_M1000GeV_W10p_proc_card.dat

Reply:
ll12 apparently is a typo and not needed, removed
as for the tt and the ww definitions, its used for the other channels looking for the same model, so prefer to keep. Else translating between cards creates too much editing problems.